docs/content/preview/yugabyte-platform/security/authentication/oidc-authentication-jumpcloud.md

```diff
@@ -0,0 +1,192 @@
+---
+title: OIDC authentication using JumpCloud in YugabyteDB Anywhere
+headerTitle: OIDC authentication with JumpCloud
+linkTitle: OIDC authentication
+description: Configuring YugabyteDB Anywhere universe to use OIDC with JumpCloud.
+headcontent: Use JumpCloud to authenticate accounts for database access
+badges: ea
+menu:
+  preview_yugabyte-platform:
+    identifier: oidc-authentication-jumpcloud
+    parent: authentication
+    weight: 30
+type: docs
+---
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+  <li>
+    <a href="../oidc-authentication-aad/" class="nav-link">
+      Azure AD
+    </a>
+  </li>
+  <li >
+    <a href="../oidc-authentication-jumpcloud/" class="nav-link active">
+      JumpCloud
+    </a>
+  </li>
+</ul>
+
+This section describes how to configure a YugabyteDB Anywhere (YBA) universe to use OIDC-based authentication for YugabyteDB YSQL database access using JumpCloud as the Identity Provider (IdP).
+
+After OIDC is set up, users can sign in to the YugabyteDB universe database using their JSON Web Token (JWT) as their password.
+
+Note that the yugabyte privileged user will continue to exist as a local database user even after OIDC-based authentication is enabled for a universe.
+
+**Learn more**
+
+- [Enable YugabyteDB Anywhere authentication via OIDC](../../../administer-yugabyte-platform/oidc-authentication/)
+- [YFTT: OIDC Authentication in YSQL](https://www.youtube.com/watch?v=KJ0XV6OnAnU&list=PL8Z3vt4qJTkLTIqB9eTLuqOdpzghX8H40&index=1)
+
+## Create an application in JumpCloud
```

Comment:
Need an intro similar to Set up OIDC with Azure AD on YugabyteDB Anywhere heading on Azure tab.
And then fix the heading levels below accordingly - this should be H3